Mixpanel/MixpanelInstance.swift

```diff
@@ -1124,14 +1122,14 @@ extension MixpanelInstance: InAppNotificationsDelegate {
 
     // MARK: - Decide
     func checkDecide(forceFetch: Bool = false, completion: @escaping ((_ response: DecideResponse?) -> Void)) {
-        networkQueue.async {
-            self.trackingQueue.sync {
-                return;
-            }
-            self.decideInstance.checkDecide(forceFetch: forceFetch,
+        self.trackingQueue.async {
+            self.networkQueue.async {
```

Comment:
i think because you flipped tracking and network in both reset and here it can still deadlock - double check this though